Userland/Utilities/file.cpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2021, Valtteri Koskivuori <vkoskiv@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/MappedFile.h>
+#include <AK/Vector.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/FileStream.h>
+#include <LibCore/MimeData.h>
+#include <LibGfx/ImageDecoder.h>
+#include <stdio.h>
+#include <unistd.h>
+
+static Optional<String> description_only(String description, [[maybe_unused]] String& path)
```

Comment:
```suggestion
static Optional<String> description_only(String description, [[maybe_unused]] const String& path)
```

(not that it really matters here :smile:)

Reply:
You know what they say - `const`sistency is key!

(sorry)